pkgs/development/python-modules/pillow-simd/default.nix

```diff
@@ -0,0 +1,79 @@
+{ stdenv, buildPythonPackage, fetchFromGitHub, isPyPy
+, olefile, freetype, libjpeg, zlib, libtiff, libwebp, tcl, lcms2
+, tk, libX11, openjpeg, libimagequant, pyroma, numpy, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "Pillow-SIMD";
+  version = "7.0.0.post3";
+
+  src = fetchFromGitHub {
+    owner = "uploadcare";
+    repo = "pillow-simd";
+    # there are no upstream tags. Grab the commit when the version in src/PIL/_version.py is bumped
+    rev = "485d2fe0779a6b0fbda7d149e2ee2a3e6bf168f3";
```

Comment:
I can see tags for e.g. v7.0.0.post3.

Also, as a general rule, you probably want the commit *immediately before* the version is bumped, because in my experience usually the version bump happens at the beginning of the dev cycle.

Reply:
You are right. I was confused by the upstream tags.